web/src/components/PoiDetails.tsx

```diff
@@ -202,6 +216,39 @@ const PoiDetails: React.FC<PoiDetailsProps> = ({
           </LinkContainer>
         )}
       </DetailSection>
+      {(website || phoneNumber || email) && (
+        <>
+          <Spacer borderColor={theme.colors.poiBorderColor} />
+          <Collapsible title={t('contactInformation')} initialCollapsed direction={direction}>
+            <>
+              {website && (
+                <ContactContainer>
+                  <Marker src={iconWebsite} alt={t('website')} />
+                  <a href={website}>
+                    {website}
+                  </a>
```

Comment:
Our `CleanLink` component should be used here instead of plain `a` tags. Also, the marker should be included in the link. If you want, you can adjust this here and for the other two usages, otherwise we'll just do that!